eoscostarica/eos-rate
# feat(footer): add footer with eoscostarica logo and share buttons

### Ticket
#71 

### Steps to test
- Open up the web app, you should see a bottom fixed navbar.

![image](https://user-images.githubusercontent.com/349542/48316694-d8d87480-e5ac-11e8-87a3-3a5f0efe9bdb.png)
